util/formatXML.xsl

```diff
@@ -0,0 +1,12 @@
+<?xml version="1.0" encoding="UTF-8"?>
+
+<xsl:stylesheet xmlns:xsl="http://www.w3.org/1999/XSL/Transform" xmlns:xs="http://www.w3.org/2001/XMLSchema" xmlns:f="http://hl7.org/fhir" xmlns:xalan="http://xml.apache.org/xslt" xmlns:xhtml="http://www.w3.org/1999/xhtml" version="2.0" exclude-result-prefixes="#all">
+    <xsl:strip-space elements="*"/>
+    <xsl:output method="xml" encoding="UTF-8" omit-xml-declaration="yes" indent="yes" xalan:indent-amount="4"/> 

```

Comment:
Geinige workaround door juist niet die factory te declareren.

Reply:
Eerlijk gezegd is dit minimaal een jaar oud en weet ik niet meer precies wat ik hier gedaan heb. Het voornaamste punt is dat je een simpele transformatie met XSLT 1.0 uitvoert waardoor je de nag-feature van de gratis versie van Saxon ontwijkt. Het bijkomende voordeel is dat al je fixtures ineens op uniforme manier geformatteerd worden waardoor je -- hopelijk -- allerlei whitespace-changes kan vermijden.